requirements_dev.txt

```
@@ -1,3 +1,4 @@
 ipython==7.31.1
 ipykernel==6.0.1
 ipython_genutils==0.2.0
+mongomock
```

If the other dependencies have a fixed version, this would also be good to have it, I suppose?
```suggestion
mongomock==4.1.2
```